### PR TITLE
replace space with _ in NC-term PTMs

### PR DESCRIPTION
**Pull request by jalew188**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -381,12 +381,12 @@ common:
   user_defined_modifications: {}
```
Comment (mschwoer) [inline README.md:381]: there's still some ` N-Term` left in `maxquant_frag_reader.ipynb`, is this on purpose?

File: peptdeep/psm_frag_reader/maxquant_frag_reader.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ def _post_process(self, mq_df):
 
             if np.any(intens > 0):
                 intens /= np.max(intens)
-            self._fragment_intensity_df.iloc[start:end, :] = intens
+            self._fragment_intensity_df.iloc[start:end, :] = np.float32(intens)
```
Comment (mschwoer) [inline peptdeep/psm_frag_reader/maxquant_frag_reader.py:141]: intentional change?

File: peptdeep/psm_frag_reader/maxquant_frag_reader.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def filter_phos(mq_df, prob):
 
     (mq_df["PhosProbs"], mq_df["PhosSites"]) = zip(
         *mq_df[["Modifications", "Phospho (STY) Probabilities"]].apply(
-            lambda x: parse_phos_probs(x[0], x[1], prob), axis=1
+            lambda x: parse_phos_probs(x.iloc[0], x.iloc[1], prob), axis=1
```
Comment (mschwoer) [inline peptdeep/psm_frag_reader/maxquant_frag_reader.py:50]: intentional change?

File: peptdeep/psm_frag_reader/maxquant_frag_reader.py (hunk 2 above)
Comment (jalew188) [reply peptdeep/psm_frag_reader/maxquant_frag_reader.py:141]: yes

File: README.md (hunk 1 above)
Comment (jalew188) [reply README.md:381]: Yes, we are waiting for alphabase's merge, then `_N-term` can pass all the tests

Comment (jalew188) [reply README.md:381]: alphabase's new release ...

**Pull request merged**